suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -26,9 +26,12 @@ import { selectDevice, selectDevices, selectDiscoveryForDevice } from '../device
 
 type ProgressEvent = BundleProgress<AccountInfo | null>['payload'];
 
-const LIMIT = 10;
+export const MAX_ACCOUNTS_PER_NETWORK = 10;
```

Comment:
This is not correct, it's max accounts per network type, no?

Reply:
You are right, the name is no ideal. But I was afraid that if the name would be `...NETWOR_TYPE`, the it might seem as refering to `NetworkType` type ("bitcoin" |"ethereum" | "cardano" | "ripple") .But after refactoring in b739e356c3, this const is not used in mobile at all anyway. So I've returned the original name to not make changes that are not relevant to this issue.